{
    "title": "An issue has been discovered in GitLab CE/EE affecting all versions starting with 12.6. Under a special condition it was possible to access data of an internal repository through a public project fork as an anonymous user.",
    "score": 7.5,
    "severity": "HIGH",
    "cwe_id": "NVD-CWE-noinfo",
    "cvss_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N",
    "cpe_match": [
        "cpe:2.3:a:gitlab:gitlab:*:*:*:*:community:*:*:*",
        "cpe:2.3:a:gitlab:gitlab:*:*:*:*:enterprise:*:*:*",
        "cpe:2.3:a:gitlab:gitlab:13.10.0:*:*:*:community:*:*:*",
        "cpe:2.3:a:gitlab:gitlab:13.10.0:*:*:*:enterprise:*:*:*",
        "cpe:2.3:a:gitlab:gitlab:*:*:*:*:community:*:*:*",
        "cpe:2.3:a:gitlab:gitlab:*:*:*:*:enterprise:*:*:*"
    ],
    "cve_source": [
        "NVD"
    ],
    "ttpMapping": []
}
